section11/numberformat_proposed.html

```diff
@@ -273,6 +228,74 @@ <h1>PartitionNumberPattern ( _numberFormat_, _x_ )</h1>
             1. Append a new Record { [[Type]]: *"unknown"*, [[Value]]: _unknown_ } as the last element of _result_.
         1. Return _result_.
       </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-getnotationsubpattern" aoid="PartitionNotationSubPattern">
+      <h1>PartitionNotationSubPattern ( _numberFormat_, _x_, _n_, _exponent_ )</h1>
+      <p>
+        The PartitionNotationSubPattern abstract operation is called with arguments _numberFormat_ (which must be an object initialized as a NumberFormat), _x_ (which is a numeric value after rounding is applied), _n_ (which is an intermediate formatted string), and _exponent_ (an integer), and creates the corresponding parts for the number and notation according to the effective locale and the formatting options of _numberFormat_. The following steps are taken:
```

Comment:
Edit: this is just an idea and the current text matches style with the current ECMA-402 text. It's probably better being addressed separately. I'm leaving the previous comment below for curiosity. Feel free to disregard the suggestion.

---


editorial nit again, maybe rephrase this to describe each of the arguments in a different paragraph? It's too much of an interruption for me reading.

```suggestion
        This operation creates the corresponding parts for the number and notation according to the effective locale and the formatting options of _numberFormat_. For each of the arguments, _numberFormat_ must be an object initialized as a NumberFormat, _x_ must be a rounded numeric value, _n_ must be an intermediate formatted result string, and _exponent_ must be an integer value.
        When the abstract operation PartitionNotationSubPattern is called with arguments _numberFormat_, _x_, _n_, and _exponent_, the following steps are taken:
```

The first paragraph could be moved to a note after the spec steps. The steps could be prefixed with `ASSERT` style steps for each value type described. WDYT?

Reply:
I modeled the intro paragraph based off the one in the existing abstract operation PartitionNumberPattern.  If we change the structure here we should change it there, too.  So for now I think we should leave the paragraph as-is.